packages/ndla-ui/src/MessageBox/MessageBox.tsx

```diff
@@ -1,39 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from '@emotion/styled';
 import Sticky from 'react-sticky-el';
 import { breakpoints, fonts, mq, spacing } from '@ndla/core';
 // @ts-ignore
 import Button from '@ndla/button';
 import { Cross } from '@ndla/icons/action';
-import { InformationOutline } from '@ndla/icons/common';
+import { InformationOutline, HumanMaleBoard } from '@ndla/icons/common';
 import { WithTranslation, withTranslation } from 'react-i18next';
 
 type WrapperProps = {
-  boxType?: 'info';
+  boxType?: 'ghost' | 'fullpage' | 'medium' | 'hide';
```

Comment:
Kunne vore definert som et enum i messagebox for å unngå å definere det samme fleire plasser.